cloud/misc/proxmox_kvm.py

```diff
@@ -745,7 +758,11 @@ def create_vm(module, proxmox, vmid, node, name, memory, cpu, cores, sockets, ti
   if module.params['api_user'] != "root@pam" and module.params['skiplock'] is not None:
     module.fail_json(msg='skiplock parameter require root@pam user. ')
 
-  taskid = getattr(proxmox_node, VZ_TYPE).create(vmid=vmid, name=name, memory=memory, cpu=cpu, cores=cores, sockets=sockets, **kwargs)
+  if module.params['clone'] == 'None':
```

Comment:
In m'y opinion, the good way is to create an function `clone_vm` and call it when `module.params['clone'] not None` ?  That way timeout can be passed to 1200 secondes by default. The task can take a while.

Note: Ensure, `state` and `clone` are mutually exclusive if clone not None and clone take the precedence.